health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreen/AppConfigurationWrapper.js

```
@@ -208,8 +245,65 @@
       )}
+      {showPopUp && (
+        <PopUp
+          type={"default"}
+          heading={t("ADD_LOCALISATION")}
+          children={[
+            <div>
+              HELLO
+              {state?.screenData
+                ?.find((i) => i.name === state?.currentScreen?.name)
+                ?.cards?.map((card) => [
+                  ...card?.fields?.map((field) => ({ message: field?.label })), // Extract label for fields
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)